aten/src/ATen/native/cuda/RreluWithNoise.cu

```diff
@@ -132,6 +133,11 @@ Tensor& rrelu_with_noise_out_cuda(const Tensor& self,
     bool training,
     c10::optional<Generator> generator,
     Tensor& output) {
+  if (self.numel() == 0) {
+    at::native::resize_output(output, self.sizes());
```

Comment:
Note: this `resize_output` line should be done before the check at the beginning of the function.